harness/determined/cli/workspace.py

```diff
@@ -146,6 +148,48 @@ def list_pools(args: argparse.Namespace) -> None:
     )
 
 
+def list_workspace_namespace_bindings(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    w = api.workspace_by_name(sess, args.workspace_name)
+    resp = bindings.get_ListWorkspaceNamespaceBindings(sess, id=w.id)
+    if resp.namespaceBindings:
+        print(f"workspace-namespace bindings for workspace {args.workspace_name}")
+        cluster_namespaces = []
+        namespace_bindings = resp.namespaceBindings
+        for cluster in namespace_bindings:
+            cluster_namespaces.append([cluster, namespace_bindings[cluster]])
+        render.tabulate_or_csv(
+            headers=["Cluster", "Namespace"],
+            values=cluster_namespaces,
+            as_csv=False,
+        )
+    else:
+        print(f"No workspace-namespace bindings for workspace {args.workspace_name}")
+    return None
+
+
+def set_workspace_namespace_binding(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    if args.cluster_name and not args.namespace:
+        raise api.errors.BadRequestException("must provide --namespace NAMESPACE")
+    w = api.workspace_by_name(sess, args.workspace_name)
+    content = bindings.v1SetWorkspaceNamespaceBindingsRequest(workspaceId=w.id)
+    cluster_name = "" if not args.cluster_name else args.cluster_name
+    content.clusterNamespacePairs = {cluster_name: args.namespace}
+    workspace_name = str(args.workspace_name)
```

Comment:
Would `workspace_name = args.workspace_name` works, or just directly use args.workspace_name since it's only referenced in one place?